### PR TITLE
Expands contribution document

### PR DESCRIPTION
**Pull request by paulodiniz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,6 +6,28 @@ agree to abide to the `Airy` [code of conduct](/code_of_conduct.md).
 Please refer to our [developers' manual](/docs/docs/developers-manual.md) to learn
 how to build, run, and test the `Airy Core Platform`.
 
+
+## Open an issue
+
+We track our development on Github using [issues](https://github.com/airyhq/airy/issues), so every Pull Request
+that is opened should refer to an issue.
```
Comment (lucapette) [inline CONTRIBUTING.md:13]: `every pull request must refer to an issue`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,6 +6,28 @@ agree to abide to the `Airy` [code of conduct](/code_of_conduct.md).
...
+
+
+## Label the issue
+
+We use labels to help track the development. If possible, label your issue as soon as it is created.
```
Comment (lucapette) [inline CONTRIBUTING.md:18]: I think good placeholder, we'll probably have to provide some guidance about how to label things (but can be done in an another pr)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,7 +45,24 @@ $ bazel test //...
...
+To keep a clean track of what is being released, every feature must contain only one commit when merged.
+The commit message of the squashed commit is very important, since it will be used to keep track of the features in a release.
+
+The convention is [#issue - description], for the example issue "Map message content" with number 216, the squashed commit message would be
+
+`[#216 - Map message content]`
+
+## Submitting
+
+When opening a Pull Request, make sure that
```
Comment (lucapette) [inline CONTRIBUTING.md:59]: `:` and the end? :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -6,6 +6,28 @@ agree to abide to the `Airy` [code of conduct](/code_of_conduct.md).
...
+## Label the issue
+
+We use labels to help track the development. If possible, label your issue as soon as it is created.
+
+## Branches
+
+Feature branches must start with `feature/*`, have the issue number and a description.
+If for example the issue is named "Map message content" and it's issue number 216, a good name would be
+
+`feature/216-map-message-content`
```
Comment (lucapette) [inline CONTRIBUTING.md:25]: I wonder if only having `feature` will confuse people. What are we supposed to do when it's a `bug` for example?

**Review by lucapette — CHANGES_REQUESTED** [review]
First pass! Amazing work, I'd go over the comments and merge as soon as possible. Can't wait to streamline this process.

I'm requesting changes as I think this document should also make sure we mention that we open PRs against the `develop` branch (so that, once this is merged, we can also switch that on)

**Review by AitorAlgorta — APPROVED** [review]

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -23,7 +49,24 @@ $ bazel test //...
...
 
-## Submit a pull request
+## Squashing your commits
+
+To keep a clean track of what is being released, every feature must contain only one commit when merged.
+The commit message of the squashed commit is very important, since it will be used to keep track of the features in a release.
+
+The convention is [#issue - description], for the example issue "Map message content" with number 216, the squashed commit message would be
+
+`[#216 - Map message content]`
```
Comment (chrismatix) [inline CONTRIBUTING.md:59]: I thought we had settled on `- [#123] Do the important thing`? 🤔

Comment (paulodiniz) [reply CONTRIBUTING.md:59]: 👍 Yes, it's way better. Will change it

**Review by lucapette — APPROVED** [review]

**Pull request merged**